tests/settings.py

```diff
@@ -82,18 +82,6 @@
     }
 ]
 
-
-# Using DatabaseCache to make sure that the cache is cleared between tests.
-# This prevents false-positives in some wagtail core tests where we are
-# changing the 'wagtail_root_paths' key which may cause future tests to fail.
-CACHES = {
-    "default": {
-        "BACKEND": "django.core.cache.backends.db.DatabaseCache",
-        "LOCATION": "cache",
-    }
-}
-
-
```

Comment:
Deleting this avoids this error when running the testapp via `./manage.py runserver` for the first time:
![image](https://github.com/torchbox-forks/wagtail-rangefilter/assets/26372762/e6cb067b-aeb5-4b46-848e-e1785cb1eaf0)

<details>
<summary>Traceback</summary>

```

Environment:


Request Method: GET
Request URL: http://0.0.0.0:8003/admin/

Django Version: 4.1
Python Version: 3.9.14
Installed Applications:
['tests.testapp',
 'wagtail_rangefilter',
 'rangefilter',
 'wagtail.contrib.search_promotions',
 'wagtail.contrib.forms',
 'wagtail.contrib.redirects',
 'wagtail.embeds',
 'wagtail.users',
 'wagtail.snippets',
 'wagtail.documents',
 'wagtail.images',
 'wagtail.search',
 'wagtail.admin',
 'wagtail.api.v2',
 'wagtail.contrib.modeladmin',
 'wagtail.contrib.routable_page',
 'wagtail.contrib.styleguide',
 'wagtail.sites',
 'wagtail',
 'taggit',
 'rest_framework',
 'django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'django.contrib.sitemaps']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'wagtail.contrib.redirects.middleware.RedirectMiddleware']



Traceback (most recent call last):
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/wagtail/models/__init__.py", line 2031, in _get_site_root_paths
    return cache_object._wagtail_cached_site_root_paths

During handling of the above exception ('Page' object has no attribute '_wagtail_cached_site_root_paths'), another exception occurred:
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/django/db/backends/sqlite3/base.py", line 357, in execute
    return Database.Cursor.execute(self, query, params)

The above exception (no such table: cache) was the direct cause of the following exception:
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/django/views/decorators/cache.py", line 62, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/wagtail/admin/urls/__init__.py", line 175, in wrapper
    return view_func(request, *args, **kwargs)
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/wagtail/admin/auth.py", line 205, in decorated_view
    return view_func(request, *args, **kwargs)
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/django/views/generic/base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/django/views/generic/base.py", line 142, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/django/views/generic/base.py", line 216, in get
    context = self.get_context_data(**kwargs)
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/wagtail/admin/views/home.py", line 294, in get_context_data
    site_details = self.get_site_details()
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/wagtail/admin/views/home.py", line 330, in get_site_details
    site = get_site_for_user(request.user)
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/wagtail/admin/navigation.py", line 33, in get_site_for_user
    root_site = root_page.get_site()
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/wagtail/models/__init__.py", line 2194, in get_site
    url_parts = self.get_url_parts()
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/wagtail/models/__init__.py", line 2066, in get_url_parts
    possible_sites = self._get_relevant_site_root_paths(request)
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/wagtail/models/__init__.py", line 2044, in _get_relevant_site_root_paths
    for srp in self._get_site_root_paths(cache_object)
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/wagtail/models/__init__.py", line 2033, in _get_site_root_paths
    cache_object._wagtail_cached_site_root_paths = Site.get_site_root_paths()
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/wagtail/models/sites.py", line 212, in get_site_root_paths
    result = cache.get("wagtail_site_root_paths")
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/django/core/cache/backends/db.py", line 53, in get
    return self.get_many([key], version).get(key, default)
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/django/core/cache/backends/db.py", line 69, in get_many
    cursor.execute(
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 103, in execute
    return super().execute(sql, params)
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/Users/kdomingo/Desktop/torchbox/forks/wagtail-rangefilter/venv/lib/python3.9/site-packages/django/db/backends/sqlite3/base.py", line 357, in execute
    return Database.Cursor.execute(self, query, params)

Exception Type: OperationalError at /admin/
Exception Value: no such table: cache
```

</details>